tests/rt.sh

```diff
@@ -402,6 +402,7 @@ else
 fi
 
 INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20201201/}
+INPUTDATA_ROOT_WW3=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20201201/}WW3_input_data_20201207/
```

Comment:
This line should be:
`INPUTDATA_ROOT_WW3=${INPUTDATA_ROOT}/WW3_input_data_20201207/`

Reply:
@DeniseWorthen Done, thanks for the suggestion.